apps/docs/pages/guides/cli/managing-environments.mdx

```diff
@@ -251,21 +255,22 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
 
     env:
       SUPABASE_ACCESS_TOKEN: ${{ secrets.SUPABASE_ACCESS_TOKEN }}
       SUPABASE_DB_PASSWORD: ${{ secrets.STAGING_DB_PASSWORD }}
-      STAGING_PROJECT_ID: abcdefghijklmnopqrst
+      SUPABASE_PROJECT_ID: ${{ secrets.STAGING_PROJECT_ID }}
```

Comment:
Is this a breaking change? as in will `STAGING_PROJECT_ID` still work or will existing projects be broken due to this change?

Reply:
This is to keep our docs up to date for onboarding new users. There's no breaking change as we don't change any existing GHA workflows that customers have setup.